content/docs/concepts/security/index.md

```diff
@@ -352,13 +355,14 @@ spec:
     methods: ["GET"]
 {{< /text >}}
 
-In `ServiceRole`, the combination of "namespace"+"services"+"paths"+"methods" defines "how a service (services) is allowed to be accessed".
-In some situations, you may need to specify additional constraints that a rule applies to. For example, a rule may only applies to a
-certain "version" of a service, or only applies to services that are labeled "foo". You can easily specify these constraints using
-custom fields.
+In a ServiceRole, the combination of “namespace”+“services”+“paths”+“methods” defines “how a service or services can be accessed”.
+In some situations, you may need to specify additional conditions for your rules. For example, a rule may only apply to a certain
+version of a service, or only apply to services that are labeled “foo”. You can easily specify these conditions using constraints.
 
-For example, the following `ServiceRole` definition extends the previous "products-viewer" role by adding a constraint on service "version"
-being "v1" or "v2". Note that the "version" property is provided by `"action.properties.version"` in "requestcontext".
+For example, the following ServiceRole definition extends the previous “products-viewer” role by adding a constraint that
+`request.headers[“version”]` is either “v1” or “v2”. Note that the supported “key” of a constraint are listed in the
+[“constraints and properties”](https://preliminary.istio.io/docs/reference/config/authorization/constraints-and-properties/) page.
```

Comment:
should use /docs/reference/config/authorization/constraints-and-properties/ instead of https://preliminary.istio.io/docs/reference/config/authorization/constraints-and-properties/